### ISSUE TITLE
Not all data points of column chart are announced when read with screen reader

### ISSUE THREAD

**Issue by franklinefrancis** [highcharts/highcharts#16539]
#### Expected behaviour
While navigating column chart with screen reader (NVDA) all data points are not announced in sequence.

#### Actual behaviour
Some data points are being skipped while navigating with NVDA.

![Column Chart NVDA Issue](https://user-images.githubusercontent.com/1227574/138825634-4a0fee4a-bbb0-4b4a-ad5a-4b3a51ac2a8a.gif)

#### Live demo with steps to reproduce
1. Run https://jsfiddle.net/franklinefrancis/x298vkzy/
2. Start NVDA
3. Try navigating data section in browse or focus mode of NVDA
4. Note that some data points are not getting announced (every 4th data point is being announced instead)

#### Product version
- HC 8.2.0

#### Affected browser(s)
- All Browsers
- NVDA 2021.2
